examples/tick-single-cluster/README.md

```diff
@@ -1,7 +1,7 @@
 # TICK Single Cluster Example
 
 This folder shows an example of Terraform code that uses the
-[tick-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/tick-cluster) 
+tick-cluster
```

Comment:
Should say and link to [influxdb-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/influxdb-cluster)